src/community/wps-download/src/test/java/org/geoserver/wps/gs/download/DownloadProcessTest.java

```diff
@@ -939,11 +955,11 @@ public void testDownloadRasterSelectedBandsScaledWithROI() throws Exception {
             // check envelope
             Assert.assertEquals(-130.88669845369998,
                     gc.getEnvelope().getLowerCorner().getOrdinate(0), 1E-6);
-            Assert.assertEquals(48.611129008700004, gc.getEnvelope().getLowerCorner()
+            Assert.assertEquals(48.5552612829, gc.getEnvelope().getLowerCorner()
```

Comment:
Can you elaborate on these test value changes? I'm guessing it's a different adjustment between having the requested bbox or the requested resolution, when the two are inconsistent, e.g, not leading to an integer number of pixels?

Reply:
Well, right now, once you specify a target size, read and resize are made by GridCoverageRenderer. Before the PR, there was different code paths involved (so a read followed by a resize).

Reply:
Ok